tests/backup/backup_restore_basic_test.go

```diff
@@ -2015,6 +2015,10 @@ var _ = Describe("{CloudSnapsSafeWhenBackupLocationDeleteTest}", func() {
 				log.InfoD("Created Cloud Credentials with name - %s", credName)
 				customBackupLocationName = fmt.Sprintf("autogenerated-backup-location-%v", time.Now().Unix())
 				customBucket = GetCustomBucketName(provider, "cloudsnapssafewhenbackuplocationdelete")
+				// Check if custom bucket is empty or not
+				result, err := IsS3BucketEmpty(customBucket)
+				dash.VerifyFatal(err, nil, "Checking for errors while checking s3 bucket")
+				dash.VerifyFatal(result, true, "Checking if bucket is empty")
```

Comment:
How about?

log.FailOnError(err, nil, "failed to check if the S3 bucket is empty") // log.FailOnError instead of dash.VerifyFatal
dash.VerifyFatal(result, true, "Checking if bucket is empty")

Reply:
@kphalgun-px Any specific reasons why you want to do this instead of 2 verfiyfatel?

Reply:
Okay will change it.